app/views/accounts/account/_credit.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="relative p-4 border border-gray-100 bg-offwhite rounded-xl focus-within:bg-white focus-within:shadow focus-within:opacity-100">
+    <label for="type" class="block text-sm font-medium opacity-50 focus-within:opacity-100">Type</label>
```

Comment:
Should use rails form help `ff.label :card_type`

And use internatiolizations...

Reply:
Thanks @ricsdeol Noted. Will do in new PR